Assets/Scripts/DogAI/DogAI.cs

```diff
@@ -0,0 +1,131 @@
+﻿/*
+ * Author(s): Timothy Ng, Isaiah Mann
+ * Description: Brains behind the dogs actions on the world
+ * Usage: [no notes]
+ */
+
+using System.Collections;
+using System;
+using UnityEngine;
+using k = PPGlobal;
+
+public class DogAI : MonoBehaviourExtended 
+{
+    DogState currentState = DogState.Idle;
+    Vector2 wanderCenter;
+
+    Vector2 target;
+
+    int tapCount = 0;
+
+    IEnumerator currentStateRoutine;
+
+    bool isActive = true;
+
+    //TODO: Replace with q system
+    float timePerState = 4f;
+
+    int tapToHeart;
+    float dogSpeed;
+
```

Comment:
Should include a `#region` tag for overrides (my bad since I wrote this):

```
#region MonoBehaviourExtended Overrides

protected override void setReferences()
{
...
}

#endregion